docs/index.md

```diff
@@ -1,37 +1,59 @@
-[Video.js homepage](http://videojs.com)
+# [Video.js][vjs] Documentation
 
-<h1>Video.js Documentation</h1>
+There are two categories of docs: [Guides][guides] and [API docs][api].
 
-There are two categories of docs: [Guides](./guides/) and [API docs](./api/). Guides explain general topics and use cases (e.g. setup). API docs are automatically generated from the codebase and give specific details about functions, properties, and events.
-
-(Corrections and additions welcome)
+Guides explain general topics and use cases (e.g. setup). API docs are automatically generated from the codebase and give specific details about functions, properties, and events.
 
 ## Guides
 
 ### Getting Started
 
-* [Setup](./guides/setup.md) - The setup documentation gives a deeper view of the additional methods you can use to trigger the player setup.
+#### [Setup][guides-setup]
 
-* [Options](./guides/options.md) - There are a number of options that can be used to change how the player behaves, starting with the HTML5 media options like autoplay and preload, and expanding to Video.js specific options.
+The setup guide covers all methods of setting up Video.js players.
 
-* [Tracks](./guides/tracks.md) - Tracks are used for displaying text information over a video, selecting different audio tracks for a video, or selected different video tracks for a video.
+#### [Options][guides-options]
 
-### Customizing
+There are a number of options that can be used to change how the player behaves, starting with the HTML5 media options like autoplay and preload, and expanding to Video.js specific options.
+
+#### [Tracks][guides-tracks]
 
-* [API](./guides/api.md) - The Video.js API allows you to control the video through javascript or trigger event listeners, whether the video is playing through HTML5, flash, or another playback technology.
+Tracks are used for displaying text information over a video, selecting different audio tracks for a video, or selecting different video tracks.
```

Comment:
if we have a video track guide, we may want to specify that video tracks are untested and may not work.